src/Catel.Core/Catel.Core.Shared/Logging/Log.cs

```diff
@@ -23,25 +23,64 @@ public class Log : ILog
         /// <summary>
         /// Initializes a new instance of the <see cref="Log"/> class.
         /// </summary>
-        /// <param name="targetType">The type for which this log is intented.</param>
-        /// <exception cref="ArgumentNullException">The <paramref name="targetType"/> is <c>null</c>.</exception>
+        /// <param name="targetType">The type for which this logger is intended.</param>
+        /// <exception cref="ArgumentNullException">If <paramref name="targetType"/> is <c>null</c>.</exception>
         public Log(Type targetType)
         {
             Argument.IsNotNull("targetType", targetType);
 
+            Name = targetType.FullName;
+            TargetType = targetType;
+
+            IsCatelLogging = targetType.IsCatelType();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Log"/> class.
+        /// </summary>
+        /// <param name="name">The name of this logger.</param>
+        /// <exception cref="ArgumentException">If <paramref name="name"/> is null or a whitespace.</exception>
+        public Log(string name)
+        {
+            Argument.IsNotNullOrWhitespace("name", name);
+
+            Name = name;
+            TargetType = null;
```

Comment:
We could also introduce a "UnknownLoggingTarget" dummy type and always set it, but not sure if that's just overhead trying to hold onto something that we shouldn't.